pyspark/bigdl/keras/converter.py

```diff
@@ -886,16 +888,6 @@ def to_bigdl_2d_padding(self, border_mode):
         else:
             raise Exception("Unsupported border mode: %s" % border_mode)
 
-    def to_bigdl_1d_padding(self, border_mode, kernel_w):
-        if border_mode == "same":
```

Comment:
why delete this?

Reply:
This is never used lol. For all 1D layers, we use 2D layers to achieve. Thus, for padding, the manner is the same as 2D.